docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,62 @@
+= Installation
+
+ifeval::["{page-component-display-version}" == "main"]
+:download-path: https://github.com/kubearchive/kubearchive/releases/latest/download
+endif::[]
+ifeval::["{page-component-display-version}" != "main"]
+:download-path: https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}
+endif::[]
+
+This document explains how to install and configure KubeArchive in your Kubernetes cluster.
+
+== Prerequisites
+
+* A PostgreSQL instance
+* CertManager is installed on the Kubernetes cluster (+v1.9.1)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.15.0)
+
+== Prepare the Database
+
+. Create a user and a password for KubeArchive to use.
+. Create a database for the user to use.
+. As the user, execute the `ddl.sql` we provide in the database, located at:
```

Comment:
I think this should either be "we provide for initializing the database" or "we provide in the `database` directory", depending on your intention.

Reply:
Done